rapids-cmake/cuda/set_architectures.cmake

```diff
@@ -89,10 +94,10 @@ function(rapids_cuda_set_architectures mode)
     list(TRANSFORM CMAKE_CUDA_ARCHITECTURES APPEND "-real")
   endif()
 
-  # CUDA 12.8.0 and later warns when compiling for arch 70. We ignore this warning when compiling
+  # For the CUDA 12.X.0 series we want to silence warnings when compiling for arch 70 when compiling
   # for RAPIDS architectures.
-  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_GREATER_EQUAL
-                                                  12.8.0)
+  if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERSION_LESS_EQUAL
```

Comment:
Why not `CMAKE_CUDA_COMPILER_VERSION VERSION_LESS 13.0.0`?

Reply:
I originally wrote this before CUDA 13 but after CUDA 12.9 was declared the last of the 12 series